### DISCUSSION TITLE
等星见雅的绳匠们集合！

### DISCUSSION THREAD

**Discussion by silverwolf510trillion** [share121/inter-knot#1077] [其他]
星见雅池子开前一发菲林不抽，特此发帖立誓！各位等星见雅的绳匠们在评论区晒出你们攒的菲林吧！
![quan](https://github.com/user-attachments/assets/b8525c24-9202-4f3b-89f7-5f05279a0c00)
（作者：quan）
![雅！](https://github.com/user-attachments/assets/566cfcf2-e6f7-4f69-aaa0-90b16267e464)

**Comment by SonwNight3089** [comment]
雅之前的角色都没有欲望抽，抽个艾连打本外就没动过：D

**Comment by silverwolf510trillion** [comment]
另外作为一名秽土转生过的神匠（上一个号因为贴了b站的链接被标记为spam了上不了绳网），给各位提两点需要注意的东西:1是回复完或是发了贴子之后记得给帖子点桑unsubscribe（取消关注），否则你的邮箱会被大量的回复轰炸。2是不要随便贴链接否则可能会被标记为spam，然后就上不了绳网了。切记！这是老一辈绳匠血的教训！

**Reply by Y-0930** [reply]
我好像看到过你b站的评论U•ェ•*U

**Reply by silverwolf510trillion** [reply]
确实，那里我也发过一边hhh

**Comment by morramu** [comment]
目前攒了227抽
![Uploading QQ图片20240810223943.png…]()

**Comment by Ropemakers5302** [comment]
一心一意只为了星见雅！！！

**Comment by yaotian-yyds** [comment]
来了

**Comment by martiansky** [comment]
awa
